components/nimbus/src/enrollment.rs

```diff
@@ -1132,9 +1163,12 @@ impl Defaults for EnrolledFeatureConfig {
     }
 }
 
-#[cfg(test)]
-impl EnrolledFeatureConfig {
-    pub fn is_rollout(&self) -> bool {
+impl SharedExperimentMetadata for EnrolledFeatureConfig {
+    fn get_slug(&self) -> String {
+        self.slug.clone()
+    }
+
+    fn is_rollout(&self) -> bool {
         self.branch.is_none()
```

Comment:
So this was previously used to determine if something was a rollout in tests, but I'm iffy on doing the same here.

Would it make more sense to put a field for `is_rollout` on the `EnrolledFeatureConfig` struct and make sure it's set properly? Not sure how big of a lift that would be or if that would result in a little bit of a nasty refactor with Swift/Kotlin.

FWIW making that adjustment can come later — the first Nimbus web adopters don't plan to hold onto the enrolled data in the browser or anywhere else, meaning `prev_experiments` will always be empty coming into the function for Nimbus web.

Reply:
`EnrolledFeatureConfig` doesn't currently cross the FFI. As alluded to in https://github.com/mozilla/application-services/pull/5475, I'm not sure it should now.

I wonder if we should have a slimmed down `prev_enrollments` with `next_experiments` which we can derive `prev_experiments`? But as you said, the client isn't going to be storing any enrolled data in the browser.

Reply:
Hm we could probably do something similar to what I did with the `ExperimentMetadata` trait? I'd have to dig through the code to double-check everything `prev_enrollments` is used for, but I do wholly agree. 

I think if we want to make that adjustment though I can make it in the UDL PR or in a later refactor (I'm leaning towards the latter)